charts/sematext-agent/README.md

```diff
@@ -64,49 +31,6 @@ $ helm uninstall st-agent
-| `region`                               | Sematext region                   | `US` Sematext US or EU region             |
-| `agent.image.repository`               | The image repository              | `sematext/agent`                          |
-| `agent.image.tag`                      | The image tag                     | `latest`                                  |
-| `agent.image.pullPolicy`               | Image pull policy                 | `Always`                                  |
-| `agent.service.port`                   | Service port                      | `80`                                      |
-| `agent.service.type`                   | Service type                      | `ClusterIP`                               |
-| `agent.resources`                      | Agent resources                   | `{}`                                      |
-| `serviceAccount.create`                | Create a service account          | `true`                                    |
-| `serviceAccount.name`                  | Service account name              | `Nil` Defaults to chart name              |
-| `priorityClassName`                    | Priority class name               | `Nil`                                     |
-| `rbac.create`                          | RBAC enabled                      | `true`                                    |
-| `tolerations`                          | Tolerations                       | `[]`                                      |
-| `nodeSelector`                         | Node selector                     | `{}`                                      |
-| `serverBaseUrl`                        | Custom Base URL                   | `Nil`                                     |
-| `logsReceiverUrl`                      | Custom Logs receiver URL          | `Nil`                                     |
-| `eventsReceiverUrl`                    | Custom Event receiver URL         | `Nil`                                     |
-| `commandServerUrl`                     | Custom Command server URL         | `Nil`                                     |
-
-Specify each parameter using the `--set key=value` argument to `helm install`. For example:
-
-```bash
-$ helm repo add sematext https://cdn.sematext.com/helm-charts/
-$ helm install st-agent \
-    --set infraToken=YOUR_INFRA_TOKEN \
-    --set region=US \
-    --set agent.image.tag=1.16.11 \
-    --set agent.image.pullPolicy=IfNotPresent \
-    sematext/sematext-agent
-```
-
-Alternatively, you can use a YAML file that specifies the values while installing the chart. For example:
-
-```bash
-$ helm repo add sematext https://cdn.sematext.com/helm-charts/
-$ helm install st-agent -f custom_values.yaml sematext/sematext-agent
-```
+For more detailed information, refer to our [official documentation](https://sematext.com/docs/agents/sematext-agent/kubernetes/helm/).
```

Comment:
official documentation ==> official Kubernetes Helm chart documentation
(so people don't think we are pointing to top level docs)

Reply:
Done